sprokit/src/sprokit/tools/tool_usage.cxx

```diff
@@ -56,18 +56,18 @@ tool_usage(int ret, boost::program_options::options_description const& options)
 void
 tool_version_message()
 {
-  std::cout << "sprokit " SPROKIT_VERSION_FULL << std::endl;
-  std::cout << "Built with sprokit: " SPROKIT_VERSION << std::endl;
+  std::cout << "KWIVER " KWIVER_VERSION_FULL << std::endl;
```

Comment:
@linus-sherrill This is one change I was uncertain about.  I probably makes sense to say "KWIVER" here because sprokit is part of KWIVER and doesn't have a different version number from KWIVER.  But I wasn't sure if this would be confusing for sprokit tools.  Then again, maybe all of this is changing anyway with KWIVER applets.  

It might be good for the new `kwiver` tool to have a top level `-v` or `--version` flag that prints out version info similar to this.

Reply:
I don't think there is a problem calling this KWIVER and I agree that this will get rolled into the tool runner as the sprokit tools are converted.